adi/assessment_data_import.py

```diff
@@ -97,17 +97,14 @@ def import_data(
 
     Returns
     -------
-    bool : Returns a boolean indicating if the asssessment data import was
+    bool : Returns a boolean indicating if the assessment data import was
     successful.
 
     """
     # Boto3 clients for S3 and SSM
     s3_client = boto3_client("s3")
     ssm_client = boto3_client("ssm")
 
-    # TODO Add error checking?
```

Comment:
Should this TODO have be deleted? There is no error checking or exception catching

Reply:
That comment was deleted in a later commit: https://github.com/cisagov/assessment-data-import-lambda/pull/1/commits/3e7491fc05db1e09b7e4abf623549096511d0822